test/unit/spec/appWidgets/input/intInputSpec.js

```diff
@@ -35,7 +39,8 @@ define(['widgets/appWidgets2/input/intInput', 'common/runtime'], (IntInput, Runt
 
         afterEach(() => {
             bus.stop();
-            window.kbaseRuntime = null;
+            runtime.destroy();
```

Comment:
Any tests that declare `runtime` in the suite also gets to keep the `runtime` so it can be `destroy`ed directly.

Reply:
was just about to ask about that!

Reply:
I wasn't super specific about it. There's a number of places that do `Runtime.make().config('blahblah')` inline in a test. That *could* be declared as a runtime object then destroyed properly, but that would've made an even larger PR...